docs/sources/tutorials/grafana-fundamentals/index.md

```diff
@@ -215,9 +215,8 @@ Dashboards consist of _panels_, each representing a part of the story you want y
 
 Every panel consists of a _query_ and a _visualization_. The query defines _what_ data you want to display, whereas the visualization defines _how_ the data is displayed.
 
-1. Click **Dashboard** in the left-side menu.
-1. On the Dashboards page, click **New** and select **New Dashboard** in the dropdown.
-1. Click **+ Add visualization**.
+1. In the sidebar, hover over the **Create** (plus sign) icon and then click **Dashboard**.
+1. Click **Add new panel**. The Prometheus data source that you added will already be selected.
```

Comment:
The DS will be selected, but the modal is opened anyway. Just in case you want to specify

![2023-06-05 17 54 18](https://github.com/grafana/grafana/assets/5699976/99e2a933-8940-41bd-920a-48edcf1a46ff)